components/nexus-cache/src/test/java/org/sonatype/nexus/cache/internal/EhCacheTrial.java

```diff
@@ -10,62 +10,65 @@
  * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
  * Eclipse Foundation. All other trademarks are the property of their respective owners.
  */
-package org.sonatype.sisu.ehcache;
-
-import java.io.File;
+package org.sonatype.nexus.cache.internal;
 
 import org.sonatype.sisu.litmus.testsupport.TestSupport;
 
+import net.sf.ehcache.Cache;
+import net.sf.ehcache.CacheManager;
+import net.sf.ehcache.Element;
+import net.sf.ehcache.config.Configuration;
+import net.sf.ehcache.config.ConfigurationFactory;
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 
-import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.Assert.assertThat;
 
 /**
- * Test for {@link CacheManagerComponentImpl}.
+ * Trails of basic EhCache.
```

Comment:
Trials are typically used for testing behaviour outside of the normal build process - see `ExportImportTrial` and others